addons/stock_account/models/stock_valuation_layer.py

```diff
@@ -29,7 +29,7 @@ class StockValuationLayer(models.Model):
     stock_valuation_layer_ids = fields.One2many('stock.valuation.layer', 'stock_valuation_layer_id')
     stock_move_id = fields.Many2one('stock.move', 'Stock Move', readonly=True, check_company=True, index=True)
     account_move_id = fields.Many2one('account.move', 'Journal Entry', readonly=True, check_company=True)
-    account_move_line_id = fields.Many2one('account.move.line', 'Invoice Line', readonly=True, check_company=True, index=True)
+    account_move_line_id = fields.Many2one('account.move.line', 'Invoice Line', readonly=True, check_company=True, index="btree_not_null")
```

Comment:
What about `account_move_id`?

Reply:
I fucked up when I saw the `account_move_line_id`. I thought this was it.

But, before r+ing the PR.
I'm not sure this one is really necessary.
I don't find any ref to `account_move_id` in the module that would be related to a constraint/unlink/whatever.

I think I should get rid of the index on `account_move_id` that I just added